### DISCUSSION TITLE
Web Interface

### DISCUSSION THREAD

**Discussion by kevorm** [UnchartedBull/OctoDash#3755] [Ideas]
I dont knwo if i should put this here or in the Q&A...
I want to pull whats being shown on my screen into a web window that i can then include in OBS as a source for streaming.

I have my octoprint running on a Pi4.  Is this an option or would it be possible to do?

**Comment by jneilliii** [comment]
OctoDash doesn't run as a server but a local application on the pi directly. You could technically run OctoDash on the computer running OBS and then attach it that way. I've seen other people use the Dashboard plugin full screen mode URL as an OBS stream source though and might be what you're looking for.

**Reply by kevorm** [reply]
ahhh ok,   i suppose i could always wrap wayland or xorg whichever one is running with vnc, then install the htmlvnc and serve it to OBS that way.   thanks for the info.

**Reply by jneilliii** [reply]
with VNC enabled on pi (raspi-config) you can definitely connect and control OctoDash directly. I have done that for debugging/development work. not familiar with OBS, but if htmlvnc is a source that you can use, it uses the same username/password as the local pi user account.
